src/node_crypto.cc

```diff
@@ -255,13 +255,53 @@ void ThrowCryptoError(Environment* env,
                       unsigned long err,  // NOLINT(runtime/int)
                       const char* default_message = nullptr) {
   HandleScope scope(env->isolate());
+  Local<String> message;
+
   if (err != 0 || default_message == nullptr) {
     char errmsg[128] = { 0 };
     ERR_error_string_n(err, errmsg, sizeof(errmsg));
-    env->ThrowError(errmsg);
+    message = String::NewFromUtf8(env->isolate(), errmsg,
+                                  v8::NewStringType::kInternalized)
+                                    .ToLocalChecked();
   } else {
-    env->ThrowError(default_message);
+    message = String::NewFromUtf8(env->isolate(), default_message,
+                                  v8::NewStringType::kInternalized)
+                                    .ToLocalChecked();
+  }
+
+  Local<Value> exception_v = Exception::Error(message);
+  CHECK(!exception_v.IsEmpty());
+  // add the openSSLErrorStack property
+  Local<Object> exception = exception_v.As<Object>();
+  Local<String> key = String::NewFromUtf8(env->isolate(), "openSSLErrorStack",
+                                          v8::NewStringType::kInternalized)
+                                            .ToLocalChecked();
+  Local<Array> errorStack = Array::New(env->isolate());
+
+  // get the error state
+  // will only add to errorStack if state has not been cleared out
+  ERR_STATE *es;
```

Comment:
merge this line with next, so declaration and initialization is done in one line